schema/index.graphql

```diff
@@ -706,7 +871,7 @@ Wishlist item
 type WishlistItem {
   id: ID!
   articleId: ID!
-  article: Article
+  article: QuantifiedArticle
```

Comment:
At least in Spryker, Intershop and Hybris whishlist items are always quantified.